src/components/button/button.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './button.css';
+
+/**
+ * Generic Kedro Button
+ */
+const Button = ({ children, onClick, size, mode }) => (
+  <span className="kedro kui-button">
+    <button
+      className={classnames(
+        'kui-button__btn',
+        `kui-button__btn--${size}`,
+        `kui-button__btn--${mode}`
```

Comment:
Just curious... what's up with the ` instead of '?

Reply:
It's called template literals similar to f-string or % in python ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Ahh cool, thank you!